web/src/views/InvestigationDetail.vue

```diff
@@ -124,7 +124,6 @@ export default defineComponent({
       await store.dispatch.fetchCurrentInvestigation(props.investigation);
       selectedDataset.value = store.state.activeDataset;
       loaded.value = true;
-      setTimeout(() => zoom(6), 2500);
```

Comment:
Since we're no longer using this `zoom` function. It can also be removed from line 106.